docs/schemas/Donor-Metadata/Donor metadata reconciliation.csv

```diff
@@ -0,0 +1,118 @@
+No,Non-Brain Specimen Collected,tissue_type_details,Tissue Type Details,Tissue Type Details,text,,,,,,,,,
+In HSWG only,,,,Subject id,,,,,,,,,Not sure why this is repeated here,
+In HSWG only,,,,Tissue Source,,Alpha,Autopsy; Surgery,,,,,,,
+In HSWG only,,,,Date of Collection,,Date,"If the date is unknown, then code:  99/99/9999",,,,,,,
+In HSWG only,,,,Age at Date of Collection (Years),,Numeric,"A valid age between 000 and 125
+2.   If Age is unknown, then code 999",,,,,,,
+,Diagnoses,clinical_brain_diagnosis,Clinical Brain Diagnosis Available,Clinical Brain Diagnosis Available,exclusive_categorical,,,,,,,,Don't know if they are using ICD10 or MONDO codes,1/#/Yes/#/|2/#/No/#/

```

Comment:
Does this field or equivalent also appear in the Neuropathology criteria/metadata that was recently discussed in the specimens WG?